deploy/osd-scc/01-osd-sre-scc.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
```

I think copying the description from the default `securitycontextconstraints.security.openshift.io/privileged` object is a good idea:
```suggestion
metadata:
  annotations:
    kubernetes.io/description: 'privileged allows access to all privileged and host
      features and the ability to run as any user, any group, any fsGroup, and with
      any SELinux context.  WARNING: this is the most relaxed SCC and should be used
      only for cluster administration. Grant with caution.'
```